openassessment/xblock/static/sass/xb/elements/_controls.scss

```diff
@@ -101,6 +101,7 @@
     color: $action-primary-color-disabled;
     border-color: $action-primary-color-disabled;
     background: $action-primary-color-disabled-back;
+    opacity: 1;
```

Comment:
Since these buttons set their colors explicitly based on disabled state, they shouldn't respect the default opacity setting.